src/__tests__/update.test.ts

```diff
@@ -135,4 +135,17 @@ describe(`update`, () => {
       }
     `);
   });
+
+  it(`should support adding comment`, () => {
+    const query = db.update(db.bar).set({ name: `Test` }).addComment(`/*This is a comment*/`);
```

Comment:
As written, this is basically an open-ended "add any string to the beginning of the query" function that is _intended_ to be used for comments, but requires the caller to pass in the right comment syntax.

I think the function should automatically handle the comment begin/end tokens.

Also, do we need the functionality to add multiple comments? I think it would be easier for this to set a single comment.

Reply:
+1 to automatically adding the comment begin/end markers.
   - Should also throw if the contents are not comment-safe.

+1 to only supporting a single comment for now.
    - And maybe allow calling it at the beginning so that the method chaining order sort of matches the output, e.g. `db.comment(...).select(...)`.

Also, how will this interact with our sqlcommenter annotations?